### PR TITLE
Add random module and remove math module

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: examples/random/random.bal [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import ballerina/io;
+import ballerina/random;
+
+public function main() {
+    // Generate a random decimal number between 0.0 and 1.0.
+    float randomDecimal = random:createDecimal();
+    io:println("Random decimal number: ", randomDecimal);
```
Comment (ThisaruGuruge) [inline examples/random/random.bal:7]:
```suggestion

    io:println("Random decimal number: ", randomDecimal);
```

File: examples/random/random.bal [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+    // Generate a random number between the given start(inclusive) and end(exclusive) values.
+    int|error randomInteger = random:createIntInRange(1, 100);
+    io:println("Random integer number in range: ", randomInteger);
```
Comment (ThisaruGuruge) [inline examples/random/random.bal:11]:
```suggestion

    io:println("Random integer number in range: ", randomInteger);
```

File: gradle.properties [hunk 3]
```diff
@@ -10,7 +10,7 @@ stdlibEncodingVersion=1.0.7-SNAPSHOT
 stdlibIoVersion=0.5.5-SNAPSHOT
 stdlibJarraysVersion=0.9.5-SNAPSHOT
 stdlibJsonutilsVersion=1.0.5-SNAPSHOT
-stdlibMathVersion=1.0.5-SNAPSHOT
+stdlibRandomVersion=1.1.0-SNAPSHOT
```
Comment (ThisaruGuruge) [inline gradle.properties:13]: Why is this `1.1.0`, not `1.0.0` ?

Comment (MadhukaHarith92) [reply gradle.properties]: Discussed and updated the version to `0.9.0`

**Review by ThisaruGuruge — APPROVED** [review]

File: gradle.properties (hunk 3 above)
Comment (ThisaruGuruge) [reply gradle.properties]: We have to update the version in the original repo too right?

Comment (MadhukaHarith92) [reply gradle.properties]: > We have to update the version in the original repo too right?

Yes, will update.

**Review by ThisaruGuruge — CHANGES_REQUESTED** [review]
Let's not merge this, since we are adding this in the `fix-stdlib-tests` branch

**Review by praneesha — APPROVED** [review]

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**